.travis.yml

```diff
@@ -8,31 +7,20 @@ cache:
   directories:
     - ~/.cache/pip
 env:
-  - TARGET: bashate
-  - TARGET: pep8
-  - TARGET: docs
-  - TARGET: docs-gnocchi.xyz
-
-  - TARGET: py27-mysql-ceph-upgrade-from-3.1
-  - TARGET: py35-postgresql-file-upgrade-from-3.1
-
-  - TARGET: py27-mysql
-  - TARGET: py35-mysql
-  - TARGET: py27-postgresql
-  - TARGET: py35-postgresql
+  - TARGETS: pep8,docs,docs-gnocchi.xyz,py27-mysql-ceph-upgrade-from-3.1,py35-postgresql-file-upgrade-from-3.1
```

Comment:
The more problematic issue is the pyXX-<indexer> jobs that take 30-40 minutes each, other are really fast.

The gain of serializing many targets in one container, it to not download each time the 500m of the ubuntu image.

I will try another PR with only the run-tests.sh hack to see.